doc/vim-lsp.txt

```diff
@@ -304,6 +305,10 @@ LspDocumentFormat					 *LspDocumentFormat*
 
 Format the entire document.
 
+LspDocumentFormatSync				     *LspDocumentFormatSync*
+
+Same as LspDocumentFormat but synchronized.
```

Comment:
```
Same as |LspDocumentFormat| but synchronous. Useful when running |autocmd| commands.

    example:
        autocmd BufWritePre <buffer> LspDocumentFormat
```

Would be good to have an example.

Reply:
That would be great. I didn't know about inline codes. Feel free to change and merge.

Reply:
Okay but will do in next time. :+1: